lib/sensors/accelerometer.cpp

```diff
@@ -38,6 +106,12 @@ std::optional<std::int16_t> Accelerometer::getRawAcceleration(const core::Axis a
   return static_cast<std::int16_t>((*high_byte << 8) | *low_byte);
 }
 
+std::int32_t Accelerometer::getAccelerationFromRawValue(const std::int16_t rawAcceleration)
```

Comment:
in `Accelerometer::read`, the values for acceleration on each axis are already `int32_t`?

Reply:
don't get what's wrong? might be missing something

Reply:
Ignore me, got my types mixed up :(